go.mod

```diff
@@ -1,6 +1,6 @@
 module sgrep
 
-go 1.22.5
+go 1.23
```

Comment:
Not sure about this change. go's current stable version is 1.22.5. You must have the unstable bleeding edge dev version?